modin/pandas/test/test_series.py

```diff
@@ -2204,7 +2204,6 @@ def test_loc(data):
     data = np.arange(100)
     modin_series = pd.Series(data, index=index).sort_index()
     pandas_series = pandas.Series(data, index=index).sort_index()
-    # Using 'fmt: skip' below as 'black' and 'flake8' can't agree on how this should be formatted
```

Comment:
why removed?

Reply:
Hm, this diff is outdated. I had an older version of `black` that wanted to delete this line. This change should be gone now.